### PR TITLE
Add a development Procfile, document in INSTALL

### PR DESCRIPTION
**Pull request by myabc**
Since introducing Webpack (commit 66f796c0), there is an additional process required to watch and rebuild Angular app assets.

A separate development `Procfile` is one solution to make things work better _out-of-the-box_, although not necessarily the solution recommended by the maintainer of foreman (see https://github.com/ddollar/foreman/issues/358#issuecomment-17556894).

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://feature-dev-procfile.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

File: Procfile.dev [hunk 1]
```diff
@@ -0,0 +1,3 @@
+web: bundle exec unicorn --config-file config/unicorn.rb --host ${HOST:="127.0.0.1"} --port ${PORT:="8080"} --env ${RAILS_ENV:="development"}
+assets: $(npm bin)/webpack --colors --watch --progress
+worker: bundle exec rake jobs:work
```
Comment (myabc) [inline Procfile.dev:3]: Two questions about the worker process:
- for a developer this process is going to be annoyingly noisy. Should we redirect stdout (i.e. `> /dev/null 2>&1`) or amend the rake task to be quieter by default?
- do we even need this process by default in development mode?

Comment (kgalli) [inline Procfile.dev:3]: @myabc I my opinion we can remove the worker at all. I believe there are only two tasks which the worker is used for: email and copy project. Both tasks are not so relevant for the daily developer work, right?

Comment (myabc) [inline Procfile.dev:3]: @kgalli I agree, but I guess a third opinion might be useful – ping @ulferts. thoughts?

**Comment by kgalli** [comment]
I would really like to remove the worker. Do you agree @ulferts.

**Comment by kgalli** [comment]
Ok I will merge this one. We can decide on what to do with the worker process later on by open a new PR.

File: Procfile.dev (hunk 1 above)
Comment (ulferts) [inline Procfile.dev:3]: @kgalli, @myabc Can't we use env variables to have the worker process started only when the developer explicitly desires it?

Comment (myabc) [inline Procfile.dev:3]: @ulferts I don't think env variables is the right way to go about it. To disable the worker by default, a developer could add a `.foreman` file to the root of the project with the following contents:

```
concurrency: web=1,assets=1,worker=0
```

Comment (myabc) [inline Procfile.dev:3]: See http://ddollar.github.io/foreman/#DEFAULT-OPTIONS

Comment (ulferts) [inline Procfile.dev:3]: Looks good, shall we add this to the INSTALL.md and .gitignore than?

Comment (kgalli) [inline Procfile.dev:3]: :+1: I like this approach. Yes let's use the `.foreman` file and update `INSTALL.md` and `.gitignore`.

Comment (myabc) [inline Procfile.dev:3]: See #2259

**Pull request merged**